Q# compilation fails if user name contains a blank space

I am trying to run the  `TeleportationSample` in `Samples > 0.Introduction folder` of `QsharpLibraries` solution. After setting the project as `Startup project`, I perform the build operation `F5`. But the build fails with error as :- 
```
1>------ Build started: Project: Microsoft.Quantum.Canon, Configuration: Debug Any CPU ------
1>No executable found matching command "dotnet-C:\Users\deepa"
1>C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\build\Microsoft.Quantum.Development.Kit.targets(17,5): error MSB3073: The command "dotnet C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\build\../tools/qsc/qsc.dll --input "AmplitudeAmplification\AmplitudeAmplification.qs" "AmplitudeAmplification\Types.qs" "AmplitudeAmplification\Utils.qs" "Arithmetic\Arithmetic.qs" "Arrays\Arrays.qs" "Asserts\ClassicalAsserts.qs" "Asserts\QuantumAsserts.qs" "Combinators\ApplyMultiControlled.qs" "Combinators\ApplyToEach.qs" "Combinators\ApplyToEachIndex.qs" "Combinators\ApplyToFirst.qs" "Combinators\ApplyToPartition.qs" "Combinators\Bind.qs" "Combinators\Compose.qs" "Combinators\Curry.qs" "Combinators\OperationPow.qs" "Combinators\RestrictToSubregister.qs" "Combinators\With.qs" "DataStructures\Pairs.qs" "DataStructures\Stack.qs" "Data\GeneratorRepresentation.qs" "Enumeration\Filter.qs" "Enumeration\Fold.qs" "Enumeration\ForAll.qs" "Enumeration\ForAny.qs" "Enumeration\Map.qs" "Enumeration\Subarray.qs" "Enumeration\Trotter.qs" "IterateThroughCartesianProduct.qs" "Math\Complex.qs" "Math\Constants.qs" "Math\Functions.qs" "Math\Random.qs" "NoOp.qs" "PhaseEstimation\Iterative.qs" "PhaseEstimation\Quantum.qs" "PhaseEstimation\Robust.qs" "PhaseEstimation\Types.qs" "Qcvv\ProcessTomography.qs" "Qecc\5QubitCode.qs" "Qecc\7QubitCode.qs" "Qecc\BitFlipCode.qs" "Qecc\KnillDistill.qs" "Qecc\Types.qs" "Qecc\Utils.qs" "QFT.qs" "Simulation\Algorithms.qs" "Simulation\PauliEvolutionSet.qs" "Simulation\Techniques.qs" "Simulation\Types.qs" "StatePreparation\StatePreparation.qs" "UnsignedIntegers.qs" "Utils\ControlledOnBitString.qs" "Utils\Multiplexer.qs" "Utils\Paulis.qs" "Utils\Predicates.qs" "Utils\Ranges.qs" "Utils\TypeConversion.qs" --references "C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\lib\netstandard2.0\Microsoft.Quantum.MetaData.dll" "C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\lib\netstandard2.0\Microsoft.Quantum.Primitives.dll" "C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\lib\netstandard2.0\Microsoft.Quantum.Simulation.Common.dll" "C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\lib\netstandard2.0\Microsoft.Quantum.Simulation.Core.dll" "C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\lib\netstandard2.0\Microsoft.Quantum.Simulation.QCTraceSimulatorRuntime.dll" "C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\lib\netstandard2.0\Microsoft.Quantum.Simulation.Simulators.dll" --outputpath obj\qsharp\src\" exited with code 1.
1>Done building project "Microsoft.Quantum.Canon.csproj" -- FAILED.
2>------ Build started: Project: TeleportationSample, Configuration: Debug Any CPU ------
2>No executable found matching command "dotnet-C:\Users\deepa"
2>C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\build\Microsoft.Quantum.Development.Kit.targets(17,5): error MSB3073: The command "dotnet C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\build\../tools/qsc/qsc.dll --input "TeleportationSample.qs" --references "D:\Repos\Quantum\Microsoft.Quantum.Canon\bin\Debug\netstandard2.0\Microsoft.Quantum.Canon.dll" "C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\lib\netstandard2.0\Microsoft.Quantum.MetaData.dll" "C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\lib\netstandard2.0\Microsoft.Quantum.Primitives.dll" "C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\lib\netstandard2.0\Microsoft.Quantum.Simulation.Common.dll" "C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\lib\netstandard2.0\Microsoft.Quantum.Simulation.Core.dll" "C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\lib\netstandard2.0\Microsoft.Quantum.Simulation.QCTraceSimulatorRuntime.dll" "C:\Users\test user\.nuget\packages\microsoft.quantum.development.kit\0.2.1802.2202-preview\lib\netstandard2.0\Microsoft.Quantum.Simulation.Simulators.dll" --outputpath obj\qsharp\src\" exited with code 1.
2>Done building project "TeleportationSample.csproj" -- FAILED.
========== Build: 0 succeeded, 2 failed, 0 up-to-date, 0 skipped ==========
```
This might be the issue of `dotnet` CLI unable to reach the complete path as the correct path is `C:/Users/test user/...`(the path that follows) but it reads till `C:\Users\test` and then fails.

I am fairly new to both Quantum computing and `dotnet` environment so any help is highly appreciated.

--- a/Interoperability/python/packages.config
+++ b/Interoperability/python/packages.config
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Quantum.Development.Kit" version="0.2.1806.1503-preview" targetFramework="netstandard2.0" />
-  <package id="Microsoft.Quantum.Canon" version="0.2.1806.1503-preview" targetFramework="netstandard2.0" />
+  <package id="Microsoft.Quantum.Development.Kit" version="0.2.1806.2802-preview" targetFramework="netstandard2.0" />
+  <package id="Microsoft.Quantum.Canon" version="0.2.1806.2802-preview" targetFramework="netstandard2.0" />
 </packages>

--- a/LibraryTests/LibraryTests.csproj
+++ b/LibraryTests/LibraryTests.csproj
@@ -23,8 +23,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
-    <PackageReference Include="Microsoft.Quantum.Xunit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
+    <PackageReference Include="Microsoft.Quantum.Xunit" Version="0.2.1806.2802-preview" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
     <PackageReference Include="xunit" Version="2.3.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />

--- a/Microsoft.Quantum.Canon/Microsoft.Quantum.Canon.csproj
+++ b/Microsoft.Quantum.Canon/Microsoft.Quantum.Canon.csproj
@@ -9,6 +9,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/AdiabaticIsing/AdiabaticIsingSample.csproj
+++ b/Samples/AdiabaticIsing/AdiabaticIsingSample.csproj
@@ -12,6 +12,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/BitFlipCode/BitFlipCode.csproj
+++ b/Samples/BitFlipCode/BitFlipCode.csproj
@@ -11,6 +11,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/DatabaseSearch/DatabaseSearchSample.csproj
+++ b/Samples/DatabaseSearch/DatabaseSearchSample.csproj
@@ -11,6 +11,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/H2SimulationCmdLine/H2SimulationSampleCmdLine.csproj
+++ b/Samples/H2SimulationCmdLine/H2SimulationSampleCmdLine.csproj
@@ -11,6 +11,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/H2SimulationGUI/H2SimulationGUI.csproj
+++ b/Samples/H2SimulationGUI/H2SimulationGUI.csproj
@@ -11,7 +11,7 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.1-beta3" />
   </ItemGroup>
 

--- a/Samples/HubbardSimulation/HubbardSimulationSample.csproj
+++ b/Samples/HubbardSimulation/HubbardSimulationSample.csproj
@@ -11,6 +11,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/IntegerFactorization/IntegerFactorization.csproj
+++ b/Samples/IntegerFactorization/IntegerFactorization.csproj
@@ -19,6 +19,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/IsingGenerators/IsingGeneratorsSample.csproj
+++ b/Samples/IsingGenerators/IsingGeneratorsSample.csproj
@@ -11,6 +11,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/IsingPhaseEstimation/IsingPhaseEstimationSample.csproj
+++ b/Samples/IsingPhaseEstimation/IsingPhaseEstimationSample.csproj
@@ -13,6 +13,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/IsingTrotterEvolution/IsingTrotterSample.csproj
+++ b/Samples/IsingTrotterEvolution/IsingTrotterSample.csproj
@@ -11,6 +11,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/Measurement/Measurement.csproj
+++ b/Samples/Measurement/Measurement.csproj
@@ -11,6 +11,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/OpenQasm/OpenQasm.csproj
+++ b/Samples/OpenQasm/OpenQasm.csproj
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 
   <ItemGroup>

--- a/Samples/PhaseEstimation/PhaseEstimationSample.csproj
+++ b/Samples/PhaseEstimation/PhaseEstimationSample.csproj
@@ -15,6 +15,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/PythonInterop/PythonInterop.csproj
+++ b/Samples/PythonInterop/PythonInterop.csproj
@@ -9,6 +9,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/PythonInterop/environment.yml
+++ b/Samples/PythonInterop/environment.yml
@@ -10,7 +10,7 @@ name: qsharp-samples
 channels:
     - conda-forge
     - pythonnet
-    - https://solidrepo.blob.core.windows.net/release/1806.1503/src/python/bin/package
+    - https://solidrepo.blob.core.windows.net/release/1806.2802/src/python/bin/package
 dependencies:
     - python>=3.6
     - numpy
@@ -21,7 +21,7 @@ dependencies:
     - matplotlib
     - ipyparallel
     - py
-    - qsharp=0.2.1806.1503
+    - qsharp=0.2.1806.2802
     - pip:
         - qinfer
         - duecredit

--- a/Samples/PythonInterop/tomography-sample.ipynb
+++ b/Samples/PythonInterop/tomography-sample.ipynb
@@ -219,7 +219,7 @@
        "                </td>\n",
        "\n",
        "                <td>\n",
-       "                    Microsoft.Quantum.Simulation.Simulators, Version=0.2.1806.1503, Culture=neutral, PublicKeyToken=40866b40fd95c7f5\n",
+       "                    Microsoft.Quantum.Simulation.Simulators, Version=0.2.1806.2802, Culture=neutral, PublicKeyToken=40866b40fd95c7f5\n",
        "                </td>\n",
        "            </tr>\n",
        "\n",
@@ -419,7 +419,7 @@
        "                    CLR type\n",
        "                </td>\n",
        "                <td>\n",
-       "                    Microsoft.Quantum.Simulation.Core.PartialUnitary`2[[Microsoft.Quantum.Simulation.Core.Qubit, Microsoft.Quantum.Simulation.Core, Version=0.2.1806.1503, Culture=neutral, PublicKeyToken=40866b40fd95c7f5],[System.ValueTuple`2[[System.Double, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089],[Microsoft.Quantum.Simulation.Core.Qubit, Microsoft.Quantum.Simulation.Core, Version=0.2.1806.1503, Culture=neutral, PublicKeyToken=40866b40fd95c7f5]], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]\n",
+       "                    Microsoft.Quantum.Simulation.Core.PartialUnitary`2[[Microsoft.Quantum.Simulation.Core.Qubit, Microsoft.Quantum.Simulation.Core, Version=0.2.1806.2802, Culture=neutral, PublicKeyToken=40866b40fd95c7f5],[System.ValueTuple`2[[System.Double, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089],[Microsoft.Quantum.Simulation.Core.Qubit, Microsoft.Quantum.Simulation.Core, Version=0.2.1806.2802, Culture=neutral, PublicKeyToken=40866b40fd95c7f5]], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]\n",
        "                </td>\n",
        "            </tr>\n",
        "\n",
@@ -429,7 +429,7 @@
        "                </td>\n",
        "\n",
        "                <td>\n",
-       "                    Microsoft.Quantum.Simulation.Core, Version=0.2.1806.1503, Culture=neutral, PublicKeyToken=40866b40fd95c7f5\n",
+       "                    Microsoft.Quantum.Simulation.Core, Version=0.2.1806.2802, Culture=neutral, PublicKeyToken=40866b40fd95c7f5\n",
        "                </td>\n",
        "            </tr>\n",
        "\n",
@@ -461,7 +461,7 @@
        "        "
       ],
       "text/plain": [
-       "<WrappedCLRObject(Microsoft.Quantum.Simulation.Core.PartialUnitary`2[[Microsoft.Quantum.Simulation.Core.Qubit, Microsoft.Quantum.Simulation.Core, Version=0.2.1806.1503, Culture=neutral, PublicKeyToken=40866b40fd95c7f5],[System.ValueTuple`2[[System.Double, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089],[Microsoft.Quantum.Simulation.Core.Qubit, Microsoft.Quantum.Simulation.Core, Version=0.2.1806.1503, Culture=neutral, PublicKeyToken=40866b40fd95c7f5]], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]) at 0x178e09d1160>"
+       "<WrappedCLRObject(Microsoft.Quantum.Simulation.Core.PartialUnitary`2[[Microsoft.Quantum.Simulation.Core.Qubit, Microsoft.Quantum.Simulation.Core, Version=0.2.1806.2802, Culture=neutral, PublicKeyToken=40866b40fd95c7f5],[System.ValueTuple`2[[System.Double, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089],[Microsoft.Quantum.Simulation.Core.Qubit, Microsoft.Quantum.Simulation.Core, Version=0.2.1806.2802, Culture=neutral, PublicKeyToken=40866b40fd95c7f5]], mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]) at 0x178e09d1160>"
       ]
      },
      "execution_count": 11,
@@ -628,7 +628,7 @@
      "text": [
       "\n",
       "Python version:                  3.6.4 | packaged by conda-forge | (default, Dec 24 2017, 10:11:43) [MSC v.1900 64 bit (AMD64)]\n",
-      "Quantum Development Kit version: 0.2.1806.1503\n",
+      "Quantum Development Kit version: 0.2.1806.2802\n",
       "\n"
      ]
     }

--- a/Samples/Qiskit/Qiskit.csproj
+++ b/Samples/Qiskit/Qiskit.csproj
@@ -19,7 +19,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 
   <ItemGroup>

--- a/Samples/ReversibleLogicSynthesis/ReversibleLogicSynthesis.csproj
+++ b/Samples/ReversibleLogicSynthesis/ReversibleLogicSynthesis.csproj
@@ -12,6 +12,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/SimpleAlgorithms/SimpleAlgorithms.csproj
+++ b/Samples/SimpleAlgorithms/SimpleAlgorithms.csproj
@@ -12,6 +12,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/SimpleIsing/SimpleIsingSample.csproj
+++ b/Samples/SimpleIsing/SimpleIsingSample.csproj
@@ -7,6 +7,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/Teleportation/TeleportationSample.csproj
+++ b/Samples/Teleportation/TeleportationSample.csproj
@@ -11,6 +11,6 @@
   </ItemGroup>
     
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
   </ItemGroup>
 </Project>

--- a/Samples/UnitTesting/Microsoft.Quantum.Canon.Tests.csproj
+++ b/Samples/UnitTesting/Microsoft.Quantum.Canon.Tests.csproj
@@ -12,8 +12,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.1503-preview" />
-    <PackageReference Include="Microsoft.Quantum.Xunit" Version="0.2.1806.1503-preview" />
+    <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.2.1806.2802-preview" />
+    <PackageReference Include="Microsoft.Quantum.Xunit" Version="0.2.1806.2802-preview" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
     <PackageReference Include="xunit" Version="2.3.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />

